tests/integration_tests/build/test_binary_size.py

```diff
@@ -13,9 +13,9 @@
 
 SIZES_DICT = {
     "x86_64": {
-        "FC_BINARY_SIZE_TARGET": 2067944,
+        "FC_BINARY_SIZE_TARGET": 2175176,
         "JAILER_BINARY_SIZE_TARGET": 1439512,
-        "FC_BINARY_SIZE_LIMIT": 2171516,
+        "FC_BINARY_SIZE_LIMIT": 2177024,
```

Comment:
This is interesting - binary getting larger while not adding additional functionality.

Reply:
I was suprised by this also. It's due to this issue: https://github.com/firecracker-microvm/firecracker/issues/2699